src/lib/components/file-browser/terra-file-browser.ts

```diff
@@ -0,0 +1,12 @@
+import { EventEmitter } from '@angular/core';
+import { TerraStorageObject } from './model/terra-storage-object';
+import { TerraBaseStorageService } from './terra-base-storage.interface';
+
+/** Interface that acts as a lightweight injection token for the {@link TerraFileBrowserComponent}. */
+export abstract class TerraFileBrowser {
```

Comment:
Die Schreibweise in Großbuchstaben für tokens fände ich auch nicht verkehrt.. Was meinst du?

Reply:
Habe mich doch dagegen entschieden. Ist ja nicht nur ein Token, sondern tatsächlich ne Klasse.

Reply:
ok passt